R/derive_param_tte.R

```diff
@@ -111,13 +111,15 @@
 #'   observation is selected. If an event is available, the event observation is
 #'   selected. Otherwise the censoring observation is selected.
 #'
-#'   Finally
-#'   1. the variables specified for `start_date` and `start_imputation_flag` are
-#'   joined from the ADSL dataset,
-#'   1. the variables as defined by the `set_values_to` parameter are added,
-#'   1. the `ADT`/`ADTM` variable is set to the maximum of `ADT`/`ADTM` and
-#'   `STARTDT`/`STARTDTM` (depending on the `create_datetime` parameter), and
-#'   1. the new observations are added to the output dataset.
+#'   Finally:
+#'   1. The variables specified for `start_date` and `start_imputation_flag` are
+#'   joined from the ADSL dataset. Only subjects in both datasets are kept,
+#'   i.e., subjects with and event or censoring and an observation in
```

Comment:
minor typo here as well, but is this statement intend to read "i.e., subjects **with an event** or censoring and an observation in" rather than "i.e., subjects** with and event** or censoring and an observation in"?

Reply:
Yes, this is ambiguous. I meant "(an event or censoring) and an observation in `dataset_adsl`".

Is the new version clearer?

Reply:
Yes, this new version is better. Thank you.